apps/chat/src/components/Chat/Chat.tsx

```diff
@@ -871,4 +871,37 @@ export const Chat = memo(() => {
     </div>
   );
 });
-Chat.displayName = 'Chat';
+ChatView.displayName = 'ChatView';
+
+export function Chat() {
+  const { t } = useTranslation(Translation.Chat);
+
+  const areSelectedConversationsLoaded = useAppSelector(
+    ConversationsSelectors.selectAreSelectedConversationsLoaded,
+  );
+  const selectedConversationsIds = useAppSelector(
+    ConversationsSelectors.selectSelectedConversationsIds,
+  );
+  const selectedConversations = useAppSelector(
+    ConversationsSelectors.selectSelectedConversations,
+  );
+  if (
+    !areSelectedConversationsLoaded &&
+    (!selectedConversations.length ||
+      selectedConversations.some((conv) => conv.status !== UploadStatus.LOADED))
+  ) {
+    return <Loader />;
+  }
+  if (
+    selectedConversations.length !== selectedConversationsIds.length ||
+    selectedConversations.some((conv) => conv.status !== UploadStatus.LOADED)
+  ) {
+    return (
+      <NotFoundEntity
+        entity={t('Conversation')}
+        additionalText="Please select another conversation."
```

Comment:
This comment for future change: additionalText={t("Please select another conversation.")}